mutters-ink-bot/src/main/java/com/rabidgremlin/mutters/bot/ink/InkBot.java

```diff
@@ -156,7 +157,7 @@ public BotResponse respond(Session session, Context context, String messageText)
     throws BotException
   {
     log.debug("===> \n session: {} context: {} messageText: {}",
-        new Object[]{ session, context, messageText });
+            session, context, messageText);

```

Comment:
Need the Object[] bit as SLF4J otherwise doesn't log correctly.... (unless they have changed this is later versions ?)

Reply:
have reverted.